demos/android/MASVS-STORAGE/MASTG-DEMO-0035/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+package_name="org.owasp.mastestapp"
+
+../../../../utils/mastg-android-backup-adb.sh $package_name
```

Comment:
Isn't `adb backup` being deprecated?
When I try to use it, I get
```
WARNING: adb backup is deprecated and may be removed in a future release
adb: backup either needs a list of packages or -all/-shared
```
According to [https://developer.android.com/about/versions/12/behavior-changes-12#adb-backup-restrictions](https://developer.android.com/about/versions/12/behavior-changes-12#adb-backup-restrictions), apps that target Android 12 (API level 31) or higher don't get their data backed up unless `android:debuggable` is set to true. I think we should mention it in the demo.